src/__fixtures__/authUsers.ts

```diff
@@ -19,3 +19,19 @@ export const testUser1: SessionUser = {
       '/C=US/O=U.S. Government/OU=DoD/OU=PKI/OU=CONTRACTOR/CN=CAMPBELL.BERNADETTE.5244446289',
   },
 }
+
+export const cmsAdmin = {
```

Comment:
Let me know if this isn't clear. The goal is to just modify `testUser1` by adding the CMS group expected.